### PR TITLE
Added basic styling, moved fetching to redux actions

### PR DESCRIPTION
**Pull request by kaiferrall**
Added basic styling, moved fetching to redux actions.

### CODE REVIEW

**Comment by kaiferrall** [comment]
Just added some css and put data control into redux state.

File: src/actions/entries.js [hunk 1]
```diff
@@ -1 +1,38 @@
...
+export const getEntries = (cognitoId, jwtToken, weekId) => async (dispatch) => {
+  try {
+    const BEARER_TOKEN = "Bearer " + jwtToken;
+    const headers = { headers: { Authorization: BEARER_TOKEN } };
+    const path = BASE_URL + `/users/${cognitoId}/journal/weeks/${weekId}`;
+    const res = await axios.get(path, headers);
+    dispatch({
+      type: SET_ENTRIES,
+      payload: res.data.Item.docBody.journal.weeks[0].entries,
+    });
```
Comment (GurmeharS) [inline src/actions/entries.js:15]: nothing wrong, but I think you might wanna add an item to the backlog to make a `Take Notes API` class that handles sending requests to our API. Will definitely make code more scalable. But I think go with this for now and add that to the backlog.

File: src/actions/entries.js [hunk 2]
```diff
@@ -1 +1,38 @@
...
+      entry: {
+        content: "",
+      },
+    };
+    const BEARER_TOKEN = "Bearer " + jwtToken;
+    const headers = { headers: { Authorization: BEARER_TOKEN } };
+    const path =
+      BASE_URL + `/users/${cognitoId}/journal/weeks/${weekId}/entries`;
+    await axios.post(path, data, headers);
+    // re-fetch entries to show newly created entry
```
Comment (GurmeharS) [inline src/actions/entries.js:33]: same as above

File: src/actions/weeks.js [hunk 3]
```diff
@@ -1,4 +1,19 @@
...
+import axios from "axios";
+
+const BASE_URL = "https://vip3e81bu0.execute-api.us-east-1.amazonaws.com/test";
+
+export const getWeeks = (cognitoId, jwtToken) => async (dispatch) => {
+  try {
+    const BEARER_TOKEN = "Bearer " + jwtToken;
+    const headers = { headers: { Authorization: BEARER_TOKEN } };
+    const path = BASE_URL + `/users/${cognitoId}/journal/weeks`;
+    const res = await axios.get(path, headers);
```
Comment (GurmeharS) [inline src/actions/weeks.js:11]: same as above, going to stop commenting this now lol

**Review by GurmeharS — APPROVED** [review]
Beauty 💯 , left some suggestions for **future** improvements.

**Review by KrishnaSolo — APPROVED** [review]

File: src/actions/entries.js (hunk 1 above)
Comment (KrishnaSolo) [reply src/actions/entries.js:15]: yeah agree with this - I might make a ticket to do this

**Pull request merged**